rasa/core/channels/facebook.py

```diff
@@ -107,11 +108,12 @@ def _is_quick_reply_message(message: Dict[Text, Any]) -> bool:
         self, text: Text, sender_id: Text, metadata: Optional[Dict[Text, Any]]
     ) -> None:
         """Pass on the text to the dialogue engine for processing."""
-
+        self.client.send_action(SenderAction(sender_action='mark_seen').to_dict(), sender_id)
         out_channel = MessengerBot(self.client)
```

Comment:
Is there a way we can use the `MessengerBot` to send these actions instead of `self.client` directly?